lib/parse/data/parseEmbedData.ts

```diff
@@ -25,10 +25,19 @@ const parseEmbedData = (config: IEmbedConfig, rssData?: IRss): IEmbedData => {
     title: rssTitle,
     link: rssShareUrl,
     image: rssImage,
+    podcast,
     itunes
   } = rssData || {};
   const { url: rssImageUrl } = rssImage || {};
   const { image: rssItunesImage, owner: rssItunesOwner } = itunes || {};
+  const { value: podcastValue } = podcast || {};
+  const podcastValueRecipient =
+    podcastValue &&
+    podcastValue.type === 'webmonetization' &&
+    podcastValue.valueRecipient?.length > 0 &&
+    podcastValue.valueRecipient[0];
```

Comment:
Just a minor optimization. I think the length check is unnecessary. Should just be able to do `podcastValue.valueRecipient?.[0]` on line 38 and remove line 37. Would resolve to either be the object we want or `undefined`.